lib/sentry/config.ex

```diff
@@ -127,6 +129,15 @@ defmodule Sentry.Config do
     get_config(:json_library, default: Jason, check_dsn: false)
   end
 
+  def log_level do
+    case get_config(:log_level, default: :warn, check_dsn: false) do
```

Comment:
Both cases return `value` so I think we can drop the `case` and the function can return directly

```elixir
get_config(:log_level, default: :warn, check_dsn: false)
```

Reply:
Done.